src/sentry/api/endpoints/project_artifact_bundle_file_details.py

```diff
@@ -36,7 +36,8 @@ def download_file_from_artifact_bundle(
             ClosesDependentFiles(fp, archive),
             content_type=headers.get("content-type", "application/octet-stream"),
         )
-        response["Content-Length"] = file_info.file_size if file_info is not None else None
+        if file_info is not None:
+            response["Content-Length"] = file_info.file_size
```

Comment:
another win for mypy, this was producing  `Content-Length: None\r\n` in the response 🙃

Reply:
noiceee